examples/ui/ui.rs

```diff
@@ -280,7 +279,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                         size: Size::new(Val::Percent(100.0), Val::Percent(100.0)),
                         position_type: PositionType::Absolute,
                         justify_content: JustifyContent::Center,
-                        align_items: AlignItems::FlexEnd,
+                        align_items: AlignItems::FlexStart,
```

Comment:
We might be able to remove this, it will probably look the same.

Reply:
This one will change the size of the logo.